moduleframework/helpers/nspawn_helper.py

```diff
@@ -68,9 +66,11 @@ def setUp(self):
         :return: None
         """
 
-        self.setModuleDependencies()
+        trans_dict["ROOT"] = self.chrootpath
+        print_info("name of CHROOT directory:", self.chrootpath)
+        if self.is_it_module:
```

Comment:
If we remove this var in previous files, I'm not sure how to escape it here.

Reply:
I don't want to remove this variable. That's also another reason why I've done that. to have it simplier to based code if it is module or not.